### PR TITLE
Fixes for conda on GHA

### PR DESCRIPTION
**Pull request by maxscheurer**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -64,6 +67,12 @@ def get_adccore_data():
 
     import AdcCore
 
+    if is_conda_build():
+        os.environ["LDFLAGS_LD"] = \
+            os.environ["LDFLAGS_LD"].replace("-dead_strip_dylibs", "")
+        os.environ["LDFLAGS"] = \
+            os.environ["LDFLAGS"].replace("-Wl,-dead_strip_dylibs", "")
```
Comment (mfherbst) [inline setup.py:74]: Ok this obviously modifies the environment for the full built. Is this exactly what is needed?

**Review by mfherbst — APPROVED** [review]
Could you bump the GHA workflow to macOS 10.15?

File: setup.py (hunk 1 above)
Comment (maxscheurer) [reply setup.py:74]: Yes, because somehow, the destructive flags otherwise get picked up again somehow.
I find this also super annoying and ugly.

Comment (mfherbst) [reply setup.py:74]: Me too, but ok. Let's leave it for the time being. Once we manage to redo our build system with the OSS libtensor we can revisit this. Can you add a TODO?

**Comment by mfherbst** [comment]
Do you want to do a dry run first or shall we directly bump and release?

**Comment by maxscheurer** [comment]
For `conda`, we don't have enough minutes left for a dry run. It's all or nothing. I added payment info to GH so that we can get this release out, no matter what 😄

**Pull request merged**